### PR TITLE
Don't let "Edit Report" button overlap error message

### PR DESCRIPTION
**Pull request by NoahCarnahan**
Diff is a bit messy because I also rearranged some logic, but the only real changes are this:
- Added div with clearfix around (floated) button
- Added margin-bottom to button

cc @orangejenny

### CODE REVIEW

**Comment by NoahCarnahan** [comment]
Looked like this before:
screen shot 2015-08-24 at 10 26 50 am

now looks like this:
screen shot 2015-08-24 at 10 26 13 am

File: corehq/apps/userreports/templates/userreports/configurable_report.html [hunk 1]
```diff
@@ -98,18 +98,20 @@
...
-            <a class="btn btn-primary" href="
-                {% url 'edit_configurable_report' domain report.report_config_id %}
-            ">{% trans "Edit Report" %}</a>
-        </div>
-    {% elif request|toggle_enabled:"REPORT_BUILDER" and report.spec.report_meta.created_by_builder %}
-        <div class="pull-right">
-            <a class="btn btn-primary pull-right" href="
-                {% url 'edit_report_in_builder' domain report.report_config_id %}
-            ">{% trans "Edit Report" %}</a>
+            <a class="btn btn-primary" style="margin-bottom: 8px" href="
```
Comment (NoahCarnahan) [inline corehq/apps/userreports/templates/userreports/configurable_report.html]: I would have preferred to use some site-wide css class for adding this margin (I chose 8px arbitrarily), but there isn't such a class that I know of. Is there a best practice for this sort of thing?

**Comment by orangejenny** [comment]
Ohhh, I get it now (screenshots, so helpful). Page-level warnings / info / errors are meant to be above page controls & content, therefore `.alert` has a bottom margin, but not a top margin. So this is really a case of awkward styling indicating that there's a problem with the page's markup: the right thing to do is to move the edit button below the error.

I didn't look closely at the templates, but maybe dividing up main_column in base_template_new.html - say, a block for alerts, and then one for page content, with sub-templates overriding the new content block instead of main_column - is the way to go.

**Comment by NoahCarnahan** [comment]
Ah good point, thanks! Moved button below alerts; removed inline style.

screen shot 2015-08-24 at 1 51 19 pm

**Comment by orangejenny** [comment]
Pedantic, but `built-warning` would fit better in the alerts block rather than the main content block.

Looks good once tests pass.

File: corehq/apps/reports_core/templates/reports_core/base_template_new.html [hunk 2]
```diff
@@ -79,7 +79,6 @@
                     }
                 },
                 error: function(jqXHR, textStatus, errorThrown) {
-                    console.log("wah waaaah");
```
Comment (benrudolph) [inline corehq/apps/reports_core/templates/reports_core/base_template_new.html]: lol

**Comment by NoahCarnahan** [comment]
Agreed. Warning moved.

**Pull request merged**